cmake/BuildLuzer.cmake

```diff
@@ -0,0 +1,54 @@
+set(LUZER_DIR ${PROJECT_BINARY_DIR}/luzer)
+set(LUZER_BUILD_DIR ${LUZER_DIR}/build)
+set(LUZER_LIBRARY_PATH ${LUZER_BUILD_DIR}/luzer)
+list(APPEND LUZER_LIBRARIES
+  ${LUZER_LIBRARY_PATH}/luzer/luzer.so
+  ${LUZER_LIBRARY_PATH}/luzer/libcustom_mutator.so
+)
+
+include(ExternalProject)
+
+list(APPEND LUZER_CMAKE_FLAGS
+  -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}
+  -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
+  -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
+  -DLUA_INCLUDE_DIR=${PROJECT_SOURCE_DIR}/third_party/luajit/src/
+  -DLUA_LIBRARIES=${LUA_LIBRARIES}
+  -DLUAJIT_FRIENDLY_MODE=ON
+  -DLUA_HAS_JIT=ON
+)
+
+ExternalProject_Add(bundled-luzer
+  GIT_REPOSITORY https://github.com/ligurio/luzer
+  GIT_TAG 6e4d95a834a9c74f8a77acb4b2bf482add70f185
+  GIT_PROGRESS TRUE
+  GIT_SHALLOW FALSE
+  SOURCE_DIR ${LUZER_DIR}/source
+  BINARY_DIR ${LUZER_BUILD_DIR}
+  TMP_DIR ${LUZER_DIR}/tmp
+  STAMP_DIR ${LUZER_DIR}/stamp
+  CONFIGURE_COMMAND
+    ${CMAKE_COMMAND} -B <BINARY_DIR> -S <SOURCE_DIR>
+      -G ${CMAKE_GENERATOR} ${LUZER_CMAKE_FLAGS}
+  BUILD_COMMAND cd <BINARY_DIR> && ${CMAKE_MAKE_PROGRAM}
```

Comment:
Nit: I suppose we may call `cmake --build` instead of `CMAKE_MAKE_PROGRAM`.
Feel free to ignore.